x-pack/plugins/fleet/server/services/epm/elasticsearch/ingest_pipeline/helpers.ts

```diff
@@ -71,3 +72,41 @@ export function rewriteIngestPipeline(
   });
   return pipeline;
 }
+
+function _mutatePipelineContentWithNewProcessor(jsonPipelineContent: any, processor: any) {
```

Comment:
Any reason this is prefixed with `_`? Makes it seem dangerous or internal?

Reply:
it's internal but I can rename it to `mutatePipelineContentWithNewProcessor` as it's not exported